devlib/utils/ssh.py

```diff
@@ -72,9 +72,15 @@ def ssh_get_shell(host, username, password=None, keyfile=None, port=None, timeou
                 raise TargetError(message.format(host))
             time.sleep(5)
 
+    conn.sendline('shopt -s checkwinsize')
```

Comment:
Since we hit similar issue I had a look at this PR, and `shopt` is a bash-specific builtin, which does not exist in the `busybox` shell AFAIK (or even zsh, which uses `set` for similar purposes). I don't think that bash feature has equivalent in all shells, so it should probably not be relied on.